unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarModifiers/AvatarModifierArea.cs

```diff
@@ -36,6 +42,8 @@ public AvatarModifierArea()
             { "DISABLE_PASSPORTS", new DisablePassportModifier() }
         };
         model = new Model();
+
+        DataStore.i.player.otherPlayers.OnRemoved += OnOtherPlayersRemoved;
```

Comment:
```suggestion
        DataStore.i.player.otherPlayers.OnRemoved += OnOtherPlayersRemoved;
```

I think that this is not symmetric with `OnDestroy`. The constructor can be called and the `OnDestroy` won't. For monobehaviours it's safer to use `Awake()` and `OnDestroy()` for cases like this.